syntax = "proto3";

package proto;

service Greeter {
  rpc SayHello (GreetRequest) returns (GreetReply) {}
  rpc SayHelloVoid (GreetRequest) returns (GreetReply) {}
}

message GreetRequest {
  string name = 1;
}

message GreetReply {
  string message = 1;
}